int32 flag_simulator
float64[] parameters
---
bool received